app/assets/src/components/views/samples/MetadataUploadModal/Instructions.jsx

```diff
@@ -43,6 +43,19 @@ class UploadInstructions extends React.Component {
               >
                 CSV template.
               </a>
+              <p>
+                {`If your entered Host Organism does not match a supported host
+              genome, we will only subtract out ERCCs and the Human genome. You can
+              read more about how to request a new genome to be added to IDseq `}
+                <a
+                  href="https://chanzuckerberg.zendesk.com/hc/en-us/articles/360034790814-How-do-I-request-a-new-host-genome-be-added-to-IDseq-"
```

Comment:
Can we remove the redundancy? Define the description and url as constants somewhere?

Reply:
Wait is this part of this PR?

Reply:
Sorry wrong branch. I have two in progress now. One min please.